kt/godot-library/src/main/kotlin/godot/core/AABB.kt

```diff
@@ -318,6 +324,20 @@ class AABB(
         }
     }
 
+    /**
+     * Returns true if the AABB is empty.
+     */
+    fun hasSurface(): Boolean {
```

Comment:
Commentaire is incorrect, it returns false if empty

Reply:
Former comment that I did not checked. Will change it.